karma.conf.js

```diff
@@ -1,70 +0,0 @@
-// Karma configuration
```

Comment:
The karma configuration is removed and it seems that browser testing is no longer happening. Please can you restore this so we can ensure that the library continues to work in supported browsers after these changes?

Reply:
I don't think browser testing is happening even before.